src/control/cmd/dmg/pool.go

```diff
@@ -88,6 +89,61 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return poolDestroy(d.log, d.conns, d.Uuid, d.Force)
 }
 
+// PoolQueryCmd is the struct representing the command to destroy a DAOS pool.
+type PoolQueryCmd struct {
+	logCmd
+	connectedCmd
+	Uuid string `long:"pool" required:"1" description:"UUID of DAOS pool to query"`
+}
+
+// Execute is run when PoolQueryCmd subcommand is activated
+func (c *PoolQueryCmd) Execute(args []string) error {
+	req := client.PoolQueryReq{
+		UUID: c.Uuid,
+	}
+
+	resp, err := c.conns.PoolQuery(req)
+	if err != nil {
+		return errors.Wrap(err, "pool query failed")
+	}
+
+	formatBytes := func(size uint64) string {
+		return bytesize.ByteSize(size).Format("%.0f", "", false)
+	}
+
+	// Maintain output compability with the `daos pool query` output.
```

Comment:
do we want a --tabulate option to print as a table to be consistent with other commands? or maybe --legacy for this format?

Reply:
I'm not sure. This output format doesn't have scalability issues as is currently. It uses the same amount of output real estate regardless of cluster size. I believe there is a push to maintain output compatibility between tools where possible. @kccain: Any thoughts on this?